CMakeLists.txt

```diff
@@ -26,11 +26,14 @@ INCLUDE(cmake/eigen.cmake)
 INCLUDE(cmake/test.cmake)
 INCLUDE(cmake/hpp.cmake)
 
+SET(PROJECT_ORG humanoid-path-planner)
```

Comment:
This could be done in SETUP_HPP_PROJECT.

Reply:
ref https://github.com/jrl-umi3218/jrl-cmakemodules/pull/184